### PR TITLE
[IOPID-1374] Update content specs definitions

### PR DESCRIPTION
**Pull request by shadowsheep1**
## Short description
This PR adds the remote ToS management feature.

## How to test
You can try this PR with this on: https://github.com/pagopa/io-app/pull/5446

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
   "api_cgn": "https://raw.githubusercontent.com/pagopa/io-backend/v13.25.1-RELEASE/api_cgn.yaml",
   "api_cgn_merchants": "https://raw.githubusercontent.com/pagopa/io-backend/v13.25.1-RELEASE/api_cgn_operator_search.yaml",
   "api_cgn_geo": "https://raw.githubusercontent.com/pagopa/io-backend/here_geoapi_integration/api_geo.yaml",
-  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.29/definitions.yml",
+  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/IOPID-1371-add-tos-config/definitions.yml",
```
Comment (Ladirico) [inline package.json:13]:
```suggestion
  "content_specs": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.30/definitions.yml"
```

**Review by Ladirico — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
...
   "api_pagopa_walletv2": "https://raw.githubusercontent.com/pagopa/io-services-metadata/1.0.29/bonus/specs/bpd/pm/walletv2.json",
```
Comment (Ladirico) [inline package.json:14]: You need to bump all version 1.0.29 in this file with version 1.0.30... I think you forgot to do that 🚀

Comment (shadowsheep1) [reply package.json]: There where, only forgot to push https://github.com/pagopa/io-dev-api-server/pull/344/commits/ce78da7b94597289cdd6b69301f7160b6a2af95e. Thanks!

**Review by Ladirico — APPROVED** [review]
LGTM!

**Pull request merged**